### PR TITLE
Add the new certificate names for dataplane and controlplane

### PR DESCRIPTION
**Pull request by davidhadas**
Adding new exported names for new certificates - one for data plane and one for control plane

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -67,8 +67,16 @@ const (
 	CertManagerCertificateClassName = "cert-manager.certificate.networking.knative.dev"
 
 	// ServingInternalCertName is the name of secret contains certificates in serving
-	// system namespace.
-	ServingInternalCertName = "knative-serving-certs"
+	// system namespace. (Deprecated, use ServingControlCertName or ServingRoutingCertName instead)
```
Comment (ReToCode) [inline pkg/config/config.go:70]: Can we use `// Deprecated: NAME is deprecated.` as syntax? Also the comment on 71 seems redundant.

**Review by ReToCode — APPROVED** [review]
/lgtm

**Review by evankanderson — COMMENTED** [review]
/approve

**Pull request merged**